notification.c

```diff
@@ -310,6 +310,33 @@ void notification_handle_button(struct mako_notification *notif, uint32_t button
 	}
 }
 
+static enum mako_touch_binding get_touch_binding(struct mako_config *config) {
```

Comment:
No need for this function, can use `config->touch_bindings.press` directly

Reply:
I introduced this for the potential future when mako has more than just one touch_binding (single-touch, double-touch, gestures). it's not necessary though, so I'll follow your suggestion.

Reply:
made press as part of get_button_binding.